src/page/ConfigPageLayout/ConfigContent.tsx

```diff
@@ -23,13 +25,24 @@ interface WithActions {
 
 type Props = BaseProps & (WithoutActions | WithActions);
 
-export function ConfigContent({ title, children, loading = false, actions }: Props) {
+export function ConfigContent({
+  title,
+  children,
+  loading = false,
+  actions,
+  ariaLoadingLabel = 'Loading...',
```

Comment:
The `CenteredSpinner` already sets a `Loading` aria-label by default when nothing is provided, so maybe we can omit this